### PR TITLE
Added material import toggle to Importer (Issue #1041)

### PR DESCRIPTION
**Pull request by Syrrr**
Implemented general material import toggle into the Mesh importer settings
Closes https://github.com/WolvenKit/WolvenKit/issues/1041

### CODE REVIEW

File: WolvenKit.Common/Model/Arguments/ImportArgs.cs [hunk 1]
```diff
@@ -101,6 +101,17 @@ public XbmImportArgs()
     /// </summary>
     public class GltfImportArgs : ImportArgs
     {
+        /// <summary>
+        /// Should a Material.Json be imported?
+        /// </summary>
+        [Category("Import Settings")]
+        [Display(Name = "Import with Material.Json")]
+        [Description("If selected materials will be updated from a Material.json file.")]
+        public bool importMaterials { get; set; } = false;
```
Comment (rfuzzo) [inline WolvenKit.Common/Model/Arguments/ImportArgs.cs:110]: small nitpick, but can this be UpperCase?

**Review by rfuzzo — CHANGES_REQUESTED** [review]

File: WolvenKit.Common/Model/Arguments/ImportArgs.cs (hunk 1 above)
Comment (Syrrr) [reply WolvenKit.Common/Model/Arguments/ImportArgs.cs:110]: the Json? 😅

Comment (rfuzzo) [reply WolvenKit.Common/Model/Arguments/ImportArgs.cs:110]: the property name :D `importMaterials `

Comment (Syrrr) [reply WolvenKit.Common/Model/Arguments/ImportArgs.cs:110]: Oh, yeah. Sure. OMW.

Comment (Syrrr) [reply WolvenKit.Common/Model/Arguments/ImportArgs.cs]: Done. Also renamed some of the other properties that were lower-case.

**Review by rfuzzo — APPROVED** [review]

**Pull request merged**